### PR TITLE
Release Engineering PR 3: Data Extraction

### PR DESCRIPTION
**Pull request by jessetilro**
**Group** Awesome Analytics Agency

In this iteration we extracted data from all selected publications. The results are included in a temporary section in our chapter to allow for them to be reviewed.

For the next and last iteration, we will perform the data synthesis and attempt to answer our research questions based on an analysis of the extracted data. During data extraction we found some new leads. Therefore we are also considering to search for some additional sources that might allow us to fill some gaps.

### CODE REVIEW

File: 06-release-engineering-analytics.Rmd [hunk 1]
```diff
@@ -308,3 +342,1284 @@ iteratively and planned for four weekly milestones.
...
+        - Involvement of product management and productive customers
+- Limitations:
+    - Not all papers that present results about RRs,
+      have "rapid release" mentioned in the abstract.
+- Future research:
+    - Systematically search for agile and lean adoption papers
+
+Notes:
+
+- Basically contains all the answers we need 😂
```
Comment (HenkGrent) [inline 06-release-engineering-analytics.Rmd]: You may want to remove this line.

File: 06-release-engineering-analytics.Rmd [hunk 2]
```diff
@@ -308,3 +342,1284 @@ iteratively and planned for four weekly milestones.
...
+- State of the practice, because case study
+
+What open challenges in release engineering does this publication mention?
+
+- Is the ecosystem [around the studied software] shrinking or expanding?
+- How have communications in the ecosystem changed over time?
+
+What research gaps does this publication contain?
+
+- More case studies are needed
```
Comment (HenkGrent) [inline 06-release-engineering-analytics.Rmd]: You may want to be a bit more specific as to towards what goal case studies are needed.

File: 06-release-engineering-analytics.Rmd [hunk 3]
```diff
@@ -308,3 +342,1284 @@ iteratively and planned for four weekly milestones.
...
+
+What open challenges in release engineering does this publication mention?
+
+- How rapid release cycles affect code integration
+
+What research gaps does this publication contain?
+
+-
+
+Are these research gaps filled by any other publications in this survey?
```
Comment (HenkGrent) [inline 06-release-engineering-analytics.Rmd]: Open unanswered question

File: 06-release-engineering-analytics.Rmd [hunk 4]
```diff
@@ -308,3 +342,1284 @@ iteratively and planned for four weekly milestones.
...
+- A meta-model for the mining of open source bases in view of gathering data that leads to
+  assessment of the quality of projects adoping the frequent release approach.
+
+What research gaps does this publication contain?
+
+-
+
+Are these research gaps filled by any other publications in this survey?
+
+-
```
Comment (HenkGrent) [inline 06-release-engineering-analytics.Rmd]: The extracted information generally looks good, nicely done. However, there may still be some gaps left open / there may be unanswered questions. You could consider moving these, think about it.

**Comment by Inventitech** [comment]
This PR broke the build:
https://travis-ci.org/saltudelft/software-analytics-book/builds/439521756

Please fix.

**Pull request merged**